apps/devhub/utils.py

```diff
@@ -146,7 +146,7 @@ def __init__(self, file_, addon=None, listed=None):
             # from the file itself.
             try:
                 addon_data = parse_addon(file_, check=False)
-            except ValidationError, form_error:
+            except ValidationError as form_error:
```

Comment:
grepping through the code we either use `as e` or `as error`, I personally prefer `as exc` but it seems `as e` wins regarding "most often used phrase"